packages/components/organisms/f-checkout/src/services/tests/mapper.test.js

```diff
@@ -121,6 +122,43 @@ describe('checkout mapper', () => {
         expect(notesRequest.length).toBe(1);
         expect(notesRequest[0].note).toBe(userNote);
     });
+
+    describe('when geo location', () => {
```

Comment:
If you structure it like below then you can `describe` each scenario better.

```javascript
describe('geoLocation')
  describe('when geolocation is present')
    it('should map the location correctly')
  describe('when geolocation is null')
    it('should map the location correctly')
```

We also might only really need one of these tests as all we're really testing is that the mapper works, one test should be fine.